src/Corax/Utils/GrowableBuffer.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using Sparrow.Server;
+namespace Corax.Utils;
+
```

Comment:
Since we know that lots of bugs happen because of growable buffers, this should go as internal in `Sparrow` or `Sparrow.Server` if we know for a fact that Raven.Client does not use growable buffers.

Reply:
Yes, I moved it into `Sparrow.Server/Utils`